lib/BloqadeExpr/src/lower.jl

```diff
@@ -220,13 +220,12 @@ end
 
 function YaoBlocks.Optimise.to_basictypes(ex::RydInteract{D}) where D
```

Comment:
Can we get a unit test to see this handle `RydInteract` with a `BoundedLattice`?

Reply:
yes, needs to be done before merging.